utils/train.py

```diff
@@ -151,15 +152,15 @@ def train(config):
                 scores = model(model_in)
                 labels = Variable(labels, requires_grad=False)
                 loss = criterion(scores, labels)
-                loss_numeric = loss.item()
                 accs.append(print_eval("dev", scores, labels, loss))
             avg_acc = np.mean(accs)
             print("final dev accuracy: {}".format(avg_acc))
             if avg_acc > max_acc:
                 print("saving best model...")
                 max_acc = avg_acc
                 model.save(config["output_file"])
-    evaluate(config, model, test_loader)
+                best_model=copy_deepcopy(model)
```

Comment:
Where is `copy_deepcopy` defined?

Reply:
copy.deepcopy(model), sorry.
import copy.deepcopy
```suggestion
                best_model=copy.deepcopy(model)
```